scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/data/TaskProActiveDataspaces.java

```diff
@@ -392,11 +392,19 @@ public void copyInputDataToScratch(List<InputSelector> inputFiles) throws FileSy
                         public Boolean call() throws FileSystemException {
                             logger.info("Copying " + finaldsfo.getRealURI() + " to " + SCRATCH.getRealURI() +
                                     "/" + finalRelativePath);
-                            SCRATCH
-                                    .resolveFile(finalRelativePath)
+                            DataSpacesFileObject finalOut = SCRATCH.resolveFile(finalRelativePath);
+                            finalOut
                                     .copyFrom(
                                             finaldsfo,
                                             FileSelector.SELECT_SELF);
+
+                            if (!finalOut.exists()) {
+                                logger.error("There was a problem during the copy of " + finaldsfo.getRealURI() + " to " + finalOut.getRealURI() +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "There was a problem during the copy of " 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)